generators/app/templates/src/app/components/CustomTextInput/components/ShowPassword/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TouchableOpacity, Image } from 'react-native';
+
+import icEyePng from '../../assets/ic_eye.png';
+import icNoeyePng from '../../assets/ic_noeye.png';
+
+import styles from './styles';
+
+const ShowPassword = ({ onShowPassword, passwordVisible }) => (
```

Comment:
`function` or `arrow function` 🤔  To be or not to be

Reply:
jaja we have to define one of

Reply:
arrow function 👼